src/css/base/typography.scss

```diff
@@ -144,56 +144,57 @@ p {
   margin-bottom: 0;
 }
 
-.content-text h1 + p, // scope to docs
+.content-text h1 + p, // scope to docs content
 .page-dek,
-.section-subhed {
-  color: $color-mediumgray;
+.section-dek {
   font-family: $font-sans;
-  font-size: $sans-s7;
+  font-size: $sans-s8;
   font-weight: 300;
   line-height: $leading-body;
   margin-bottom: 0;
   margin-top: $grid-2;
   padding-bottom: $grid-2;
 }
 
-.section-subhed {
-  color: $color-mediumgray;
+// scope to docs content
+.content-text h1 + p {
+  font-size: $sans-s7;
 }
 
 //spacing after block elements
-p, ul, ol, pre {
+p, ul, ol, pre, div {
   & + h1,
   & + h2,
   & + h3,
   & + h4,
   & + h5 {
-    margin-top: $grid-4;
-  }
-
-  & + h2 {
-    margin-top: $grid-2;
+    margin-top: $grid-6;
   }
 }
 
-// constrain to docs content
-.content-text p,
-.content-text li {
-  a {
-    border-bottom: 1px solid;
-    padding-bottom: 2px;
-    text-decoration: none;
+// scope to docs content
+.content-text {
+  p,
+  li {
+    a {
+      border-bottom: 1px solid;
+      padding-bottom: 2px;
+      text-decoration: none;
 
-    &:visited {
-      border-bottom-style: dotted;
+      &:visited {
+        border-bottom-style: dotted;
+      }
```

Comment:
We'll let the nsa slurp all our personal communication out of telecom servers, but not put a bordered bottom rule on a `:visited` pseudoelement for privacy reasons.